Pipfile

```diff
@@ -3,7 +3,7 @@ python_version = "3.8"
 
 [packages]
 connexion = {extras = ["swagger-ui"],version = "~=2.7"}
-kerlescan = {editable = true, ref = "0.74", git = "https://github.com/RedHatInsights/kerlescan.git"}
+kerlescan = {editable = true, ref = "DRFT-492", git = "https://github.com/jpramos123/kerlescan.git"}
```

Comment:
Looks like you missed reverting this after testing?

Reply:
I was waiting Kerlescan merge so I can change the repository and tag